web/template/layout/pagination.gotmpl

```diff
@@ -0,0 +1,30 @@
+{{ define "pagination" }}
+  <nav class="moj-pagination" id="pagination-label">
+    <p class="govuk-visually-hidden" aria-labelledby="pagination-label">Pagination navigation</p>
+    <p class="moj-pagination__results">Showing <b>{{ .Start }}</b> to <b>{{ .End }}</b> of <b>{{ .TotalItems }}</b> cases</p>
+
+    {{ if ne 1 .TotalPages }}
+      <ul class="moj-pagination__list">
+        {{ if .HasPrevious }}
+          <li class="moj-pagination__item  moj-pagination__item--prev">
+            <a class="moj-pagination__link" href="?page={{ .PreviousPage }}">Previous<span class="govuk-visually-hidden"> set of pages</span></a>
+          </li>
+        {{ end }}
+
+        {{ range .Pages }}
+          {{ if eq . $.CurrentPage }}
+            <li class="moj-pagination__item moj-pagination__item--active">{{ . }}</li>
```

Comment:
Recent design system accessibility audit pointed out that links which are just numbers are unhelpful for screen readers. NB: `TotalPages` needs to be defined on `Pagination`
```suggestion
            <li class="moj-pagination__item moj-pagination__item--active"><span class="govuk-visually-hidden">Page </span>{{ . }}<span class="govuk-visually-hidden"> of {{ .TotalPages }}</span></li>
```